README.md

```diff
@@ -169,6 +169,7 @@ For more information on the structure of the YAML configuration file, please vis
 | `containerAppName`        | No       | The name of the Container App that will be created or updated. If not provided, this value will be `github-action-container-app-<github-run-id>-<github-run-attempt>`. |
 | `resourceGroup`           | No       | The existing resource group that the Azure Container App will be created in. If not provided, this value will be `<container-app-name>-rg` and its existence will first be checked before attempting to create it. |
 | `containerAppEnvironment` | No       | The name of the Container App environment to use with the application. If not provided, an existing environment in the resource group of the Container App will be used, otherwise, an environment will be created in the formation `<container-app-name>-env`. |
+| `containerAppEnvironmentResourceGroup` | No       | The name of the Resource Group that contains the Azure Container App environment to use with the application. If not provided, it will fall back to the valie of resourceGroup, which defaults to `<container-app-name>-env`. |

```

Comment:
```suggestion
| `containerAppEnvironmentResourceGroup` | No       | The name of the Resource Group that contains the Azure Container App environment to use with the application. If not provided, it will fall back to the value of resourceGroup, which defaults to `<container-app-name>-env`. |
```